python_training/sim_city.py

```diff
@@ -0,0 +1,207 @@
+BASE_MUNICIPALITY_TAX_RATE = 1000
+
+
+class Ackland:
+    """
+    The state Ackland itself.
+    """
+
+    def __init__(self, cities):
+        """
+        Initiates the Ackland state object.
+        """
+        if isinstance(cities, City):
+            cities = [cities]
+        self.cities = cities
+
+    def calculate_total_tax_amount(self) -> int:
+        """
+        Calculates the amount of tax money that would be earned from all the cities of Ackland.
+
+        :return: The total amount of tax money from all the cities of the state.
+        """
+        return sum([city.how_much_money() for city in self.cities])
+
+
+class City:
+    """
+    A city in Ackland.
+    """
+
+    def __init__(self, city_name):
+        """
+        Initiates a new City object.
+
+        :param city_name: The name of the city you would like to create.
+        """
+        self.city_name = city_name
```

Comment:
It's better not to use the class name as a prefix to its members' names.
`my_city.name` <-> `my_city.city_name`